CONTRIBUTING.md

```diff
@@ -80,9 +80,16 @@ for reference.
 
 ## Building the Project Locally
 
-Amoro is built using Maven with Java 1.8 and Java 17(only for `amoro-mixed-format/amoro-mixed-format-trino` module).
+Amoro is built using Maven with JDK 8 and JDK 17(only for `amoro-mixed-format/amoro-mixed-format-trino` module).
```

Comment:
This duplicates the Build section in README.md, can we just add a link to the documentation? This would avoid maintaining two identical sections

Reply:
Seems to be a good suggestion, we can link to https://github.com/apache/amoro?tab=readme-ov-file#building

Reply:
I have updated the doc, PTAL.